client/components/notice/style.scss

```diff
@@ -32,6 +32,10 @@
 	&.is-info {
 		background: $blue-wordpress;
 	}
+
+	&.hidden {
```

Comment:
This doesn't seem a great way to handle these — we should just not render the notice.